### PR TITLE
Simplified image widget logic

### PR DESCRIPTION
**Pull request by spoorthyv**
Simplified the image widget logic. Didn't change any functionality at all.

### CODE REVIEW

File: src/default-theme/imagewidget.less [hunk 1]
```diff
@@ -6,6 +6,9 @@
   overflow: auto;
   padding: 0 4px 4px 4px;
 }
+.jp-ImageWidget > div {
```
Comment (blink1073) [inline src/default-theme/imagewidget.less]: @ellisonbg, this will surely conflict with #802.

**Comment by blink1073** [comment]
Hi @spoorthyv, this will need a rebase now that #802 is merged.

**Comment by ellisonbg** [comment]
@spoorthyv can you rebase?

**Comment by spoorthyv** [comment]
Moved to #862

**Pull request closed without merging**